### ISSUE TITLE
[Bookworm] Fix tacplus build for Bookworm

### ISSUE THREAD

**Issue by saiarcot895** [sonic-net/sonic-buildimage#16740]
#### Description

In the bookworm branch, libnss-tacplus build has been disabled due to build errors. In addition, libpam-tacplus and audisp-tacplus installation into the base image has been disabled as there are other issues with that.

#### Steps to reproduce the issue:
1.
2.
3.

#### Output of `show version`:

```
(paste your output here)
```
#### Output of `show techsupport`:

```
(paste your output here or download and attach the file here )
```

**Comment by saiarcot895** [comment]
Compilation is fixed in #16831, but there are still tacplus errors, as reported in #16850.

**Comment by saiarcot895** [comment]
Fixed in 7ee84b7a4971e83563aa3cfd048edad388cf6396
